apps/backend/src/modules/users/user.service.ts

```diff
@@ -1,22 +1,25 @@
-import { UserError } from "~/libs/exceptions/exceptions.js";
-import { type Encryption } from "~/libs/modules/encryption/libs/types/types.js";
+import { ExceptionMessage } from "~/libs/enums/enums.js";
+import { type BaseEncryption } from "~/libs/modules/encryption/encryption.js";
 import { HTTPCode } from "~/libs/modules/http/http.js";
 import { type Service } from "~/libs/types/types.js";
+
+import { UserError } from "./libs/exceptions/exceptions.js";
 import {
 	type UserAuthResponseDto,
 	type UserGetAllResponseDto,
 	type UserSignUpRequestDto,
-} from "~/modules/users/libs/types/types.js";
-import { UserEntity } from "~/modules/users/user.entity.js";
-import { type UserRepository } from "~/modules/users/user.repository.js";
-
-import { ExceptionMessage } from "./libs/enums/enums.js";
+} from "./libs/types/types.js";
+import { UserEntity } from "./user.entity.js";
+import { type UserRepository } from "./user.repository.js";
 
 class UserService implements Service {
-	private encryption: Encryption;
+	private encryption: BaseEncryption;
```

Comment:
we should use `Encryption`, we need to export it from  `~/libs/modules/encryption/encryption.js`

```suggestion
	private encryption: Encryption;
```